037-kafka-canary-roadmap.md

```diff
@@ -0,0 +1,127 @@
+# Kafka Canary roadmap
+
+This proposal describes a possible future set of improvements and features for the [Strimzi Canary](https://github.com/strimzi/strimzi-canary) tool, which is used to check that Kafka clusters are operating correctly.  
```

Comment:
It should not be a _possible future set_. You are proposing it, so the assumption should be that if it is approved it is done. If you are not sure about it (but I don't think so), then it needs to be reevaluated.